### DISCUSSION TITLE
walletconnectKotlinV2 transaction

### DISCUSSION THREAD

**Discussion by habasa** [WalletConnect/walletconnect-monorepo#3093] [Developer Support]
Is it possible to implement Kotlin transaction(sendTransaction) with WalletconnectKotlinV2??

**Comment by crypblizz8** [comment]
@TalhaAli00

**Comment by Talhaali00** [comment]
@habasa 
Can you please provide some more information? Are you asking if you can call sendTransaction to an RPC provider?

**Reply by habasa** [reply]
hello! @TalhaAli00 

 I wonder if WalletConnectKotlinV2 basically provides send transaction function.

Currently, my situation is connected to Metamask.

After that, I'm trying to implement the transaction function, but I can't find a function related to this in the walletconnectV2 (android) document.

The function I want to implement is like the picture below.


![Screenshot_20230712-101745_MetaMask](https://github.com/WalletConnect/walletconnect-monorepo/assets/73059210/c3d06007-0404-41fd-9215-863e072a18ac)

Is it possible to implement the same function using an rpc provider?
